go/vt/orchestrator/inst/instance_dao.go

```diff
@@ -984,36 +984,36 @@ func readInstanceRow(m sqlutils.RowMap) *Instance {
 	instance.UsingMariaDBGTID = m.GetBool("mariadb_gtid")
 	instance.SelfBinlogCoordinates.LogFile = m.GetString("binary_log_file")
 	instance.SelfBinlogCoordinates.LogPos = m.GetInt64("binary_log_pos")
-	instance.ReadBinlogCoordinates.LogFile = m.GetString("master_log_file")
-	instance.ReadBinlogCoordinates.LogPos = m.GetInt64("read_master_log_pos")
-	instance.ExecBinlogCoordinates.LogFile = m.GetString("relay_master_log_file")
-	instance.ExecBinlogCoordinates.LogPos = m.GetInt64("exec_master_log_pos")
+	instance.ReadBinlogCoordinates.LogFile = m.GetString("primary_log_file")
+	instance.ReadBinlogCoordinates.LogPos = m.GetInt64("read_primary_log_pos")
+	instance.ExecBinlogCoordinates.LogFile = m.GetString("relay_primary_log_file")
+	instance.ExecBinlogCoordinates.LogPos = m.GetInt64("exec_primary_log_pos")
 	instance.IsDetached, _ = instance.ExecBinlogCoordinates.ExtractDetachedCoordinates()
 	instance.RelaylogCoordinates.LogFile = m.GetString("relay_log_file")
 	instance.RelaylogCoordinates.LogPos = m.GetInt64("relay_log_pos")
 	instance.RelaylogCoordinates.Type = RelayLog
 	instance.LastSQLError = m.GetString("last_sql_error")
 	instance.LastIOError = m.GetString("last_io_error")
-	instance.SecondsBehindMaster = m.GetNullInt64("seconds_behind_master")
-	instance.ReplicationLagSeconds = m.GetNullInt64("slave_lag_seconds")
+	instance.SecondsBehindPrimary = m.GetNullInt64("seconds_behind_primary")
+	instance.ReplicationLagSeconds = m.GetNullInt64("replica_lag_seconds")
```

Comment:
Are these two independent values or basically the same thing?

Reply:
They are not. `seconds_behind_master` is output from `show slave status`, and is always there.
`ReplicationLagSeconds` is:
- computed by `ReplicationLagQuery` if present, or
- fallback to `seconds_behind_master`
it's good to have both values.